tests/tck/features/mutate/InsertWithTimeType.feature

```diff
@@ -107,6 +107,76 @@ Feature: Insert with time-dependent types
     Then the result should be, in any order:
       | edge_date._src | edge_date._dst | edge_date._rank | edge_date.f_date | edge_date.f_time  | edge_date.f_datetime         |
       | 'test_src'     | 'test_dst'     | 0               | '2017-03-04'     | '23:01:00.000000' | '2017-03-04T22:30:40.000000' |
+    # insert with timezone offset and microseconds
+    When try to execute query:
+      """
+      INSERT VERTEX tag_date(f_date, f_time, f_datetime) VALUES "test":(date("2017-03-04"), time("23:01:00.010000+01:00"), datetime("2017-03-04T22:30:40.003000-02:30"));
+      INSERT EDGE edge_date(f_date, f_time, f_datetime) VALUES "test_src"->"test_dst":(date("2017-03-04"), time("23:01:00.010000+01:00"), datetime("2017-03-04T22:30:40.003000-02:30"));
+      """
+    Then the execution should be successful
+    When executing query:
+      """
+      FETCH PROP ON tag_date "test" YIELD tag_date.f_date, tag_date.f_time, tag_date.f_datetime;
+      """
+    Then the result should be, in any order:
+      | VertexID | tag_date.f_date | tag_date.f_time   | tag_date.f_datetime          |
+      | 'test'   | '2017-03-04'    | '00:01:00.010000' | '2017-03-04T20:00:40.003000' |
```

Comment:
expect should be `2017-03-05T01:00:40.003000`

Reply:
Fixed.